[Bug] test_pred_submitter_manager.py::test_submit_prediction_and_payout fails

I was running CI for a completely unrelated test, and I got this error.

CI log: https://github.com/oceanprotocol/pdr-backend/actions/runs/9414022639/job/25932006578?pr=1136

Traceback:
```text
=================================== FAILURES ===================================
______________________ test_submit_prediction_and_payout _______________________

pred_submitter_mgr = <pdr_backend.contract.pred_submitter_mgr.PredSubmitterMgr object at 0x7fbd1c75a3d0>
web3_config = <pdr_backend.util.web3_config.Web3Config object at 0x7fbdd395df50>
feed_contract1 = <pdr_backend.contract.feed_contract.FeedContract object at 0x7fbcfde4cd90>
feed_contract2 = <pdr_backend.contract.feed_contract.FeedContract object at 0x7fbcfdc1a1d0>
OCEAN = <pdr_backend.contract.token.Token object at 0x7fbcfdc41190>

    def test_submit_prediction_and_payout(
        pred_submitter_mgr: PredSubmitterMgr,
        web3_config,
        feed_contract1: FeedContract,
        feed_contract2,
        OCEAN,
    ):
        # the user approves 100 OCEAN tokens to the prediction manager
        OCEAN.approve(pred_submitter_mgr.contract_address, Wei(100), web3_config.owner)
    
        # get the next prediction epoch
        current_epoch = feed_contract1.get_current_epoch_ts()
    
        # set prediction epoch
        prediction_epoch = UnixTimeS(current_epoch + S_PER_EPOCH * 2)
    
        # get the OCEAN balance of the owner before submitting
        bal_before = OCEAN.balanceOf(web3_config.owner)
        assert bal_before > Wei(100), "Not enough balance to execute the test"
    
        feed_addrs = [
            feed_contract1.contract_address,
            feed_contract2.contract_address,
        ]
        # give allowance
        tx_receipt = pred_submitter_mgr.approve_ocean(feed_addrs)
        assert tx_receipt.status == 1, "Transaction failed"
    
        # submit prediction
        # first feed up 20, down 30
        # second feed up 40, down 10
        tx_receipt = pred_submitter_mgr.submit_prediction(
            stakes_up=[Wei(20), Wei(30)],
            stakes_down=[Wei(40), Wei(10)],
            feed_addrs=feed_addrs,
            epoch=prediction_epoch,
            wait_for_receipt=True,
        )
>       assert tx_receipt.status == 1, "Transaction failed"
E       AssertionError: Transaction failed
E       assert 0 == 1
E        +  where 0 = AttributeDict({'transactionHash': HexBytes('0x8221432471e5b405be19686572385984b6d6133d150d4015ed872917c9c8aff1'), 'tra...000000000000000000000000000000000000000000000000000000000000000000000000000000'), 'status': 0, 'effectiveGasPrice': 0}).status

pdr_backend/pred_submitter/test/test_pred_submitter_manager.py:165: AssertionError
=============================== warnings summary ===============================
system_tests/test_get_traction_info_system.py::test_traction_info_system
  /opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/kaleido/scopes/base.py:[188](https://github.com/oceanprotocol/pdr-backend/actions/runs/9414022639/job/25932006578?pr=1136#step:8:189): DeprecationWarning:
  
  setDaemon() is deprecated, set the daemon attribute instead

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pdr_backend/pred_submitter/test/test_pred_submitter_manager.py::test_submit_prediction_and_payout - AssertionError: Transaction failed
assert 0 == 1
 +  where 0 = AttributeDict({'transactionHash': HexBytes('0x8221432471e5b405be[196](https://github.com/oceanprotocol/pdr-backend/actions/runs/9414022639/job/25932006578?pr=1136#step:8:197)86572385984b6d6133d150d4015ed872917c9c8aff1'), 'tra...000000000000000000000000000000000000000000000000000000000000000000000000000000'), 'status': 0, 'effectiveGasPrice': 0}).status
============= 1 failed, 590 passed, 1 warning in 303.04s (0:05:03) =============
Error: Process completed with exit code 1.
```

--- a/pdr_backend/pred_submitter/test/test_pred_submitter_manager.py
+++ b/pdr_backend/pred_submitter/test/test_pred_submitter_manager.py
@@ -138,7 +138,7 @@ def test_submit_prediction_and_payout(
     current_epoch = feed_contract1.get_current_epoch_ts()
 
     # set prediction epoch
-    prediction_epoch = UnixTimeS(current_epoch + S_PER_EPOCH * 2)
+    prediction_epoch = UnixTimeS(current_epoch + S_PER_EPOCH * 3)
 
     # get the OCEAN balance of the owner before submitting
     bal_before = OCEAN.balanceOf(web3_config.owner)
@@ -170,7 +170,7 @@ def test_submit_prediction_and_payout(
 
     # fast forward time to get payout
     web3_config.w3.provider.make_request(
-        RPCEndpoint("evm_increaseTime"), [S_PER_EPOCH * 4]
+        RPCEndpoint("evm_increaseTime"), [S_PER_EPOCH * 5]
     )
     web3_config.w3.provider.make_request(RPCEndpoint("evm_mine"), [])
 

